File: 4.Sum of numbers in a given range/sum of numbers in a given range.java
import java.util.Scanner;
public class MyClass {
    public static void main(String args[])
    {
        int s,e,sum=0;
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the start Range =");
        s=sc.nextInt();
        System.out.println("Enter the End Range =");
        e=sc.nextInt();
        for (int i=s;i<=e;i++)
        {
            sum=sum+i;
        }
        System.out.println("Sum of Range "+s+" to "+e+" is = "+sum);
    }
}

File: 4.Sum of numbers in a given range/sum of numbers in a given range.c
#include<stdio.h>

int main() {
    int n,m,sum=0,i=0;
    printf("Enter the Start range =");
    scanf("%d",&n);
    printf("Enter the End range = ");
    scanf("%d",&m);
    for( i=n;i<=m;i++)
    {
        sum=sum+i;
    }
    printf("Sum of range is %d ",sum);
}
